src/Websites/Websites/Cmdlets/AppServicePlans/NewAzureAppServicePlan.cs

```diff
@@ -35,8 +35,8 @@ public class NewAzureAppServicePlanCmdlet : AppServicePlanBaseCmdlet
         [ValidateNotNullOrEmpty]
         public string Location { get; set; }
 
-        [Parameter(Position = 3, Mandatory = false, HelpMessage = "The App Service plan tier. Allowed values are [Free|Shared|Basic|Standard|Premium|PremiumV2]")]
-        [PSArgumentCompleter("Free", "Shared", "Basic", "Standard", "Premium", "PremiumV2", "Isolated", "PremiumContainer")]
+        [Parameter(Position = 3, Mandatory = false, HelpMessage = "The App Service plan tier. Allowed values are [Free|Shared|Basic|Standard|Premium|PremiumV2|PremiumV3]")]
+        [PSArgumentCompleter("Free", "Shared", "Basic", "Standard", "Premium", "PremiumV2", "PremiumV3", "Isolated", "PremiumContainer")]
```

Comment:
>, "PremiumContainer" [](start = 115, length = 20)

we won't GA PremiumContainer SKU and we should remove it

Reply:
Yes. Will remove it in next PR.